libs/designer-ui/src/lib/tokenpicker/index.tsx

```diff
@@ -264,3 +266,12 @@ export function getWindowDimensions() {
     height,
   };
 }
+
+export function isTokenPickerElement(element: HTMLElement | null) {
+  for (let e = element; e !== null; e = e.parentElement) {
+    if (e.className.includes(tokenPickerClassName)) {
```

Comment:
can we change this to:
`
if (e.className?.includes(tokenPickerClassName)) {
`
it will crash for elements without classNames

Reply:
Thanks for the catch !